anda/langs/go/act/golang-github-nektos-act.spec

```diff
@@ -0,0 +1,88 @@
+# Generated by go2rpm 1.15.0
+%bcond check 1
+%bcond bootstrap 0
+
+%if %{with bootstrap}
+%global debug_package %{nil}
+%endif
+
+%if %{with bootstrap}
+%global __requires_exclude %{?__requires_exclude:%{__requires_exclude}|}^golang\\(.*\\)$
+%endif
+
+# https://github.com/nektos/act.git
+%global goipath         github.com/nektos/act.git
+Version:                0.2.75
+
+# REMOVE BEFORE SUBMITTING THIS FOR REVIEW
+# ---
+# New Fedora packages should use %%gometa -f, which makes the package
+# ExclusiveArch to %%golang_arches_future and thus excludes the package from
+# %%ix86. If the new package is needed as a dependency for another package,
+# please consider removing that package from %%ix86 in the same way, instead of
+# building more go packages for i686. If your package is not a leaf package,
+# you'll need to coordinate the removal of the package's dependents first.
+# ---
+# REMOVE BEFORE SUBMITTING THIS FOR REVIEW
+%gometa -L -f
+
+%global common_description %{expand:
+# FIXME}
+
+%global golicenses      LICENSE pkg/lookpath/LICENSE
+%global godocs          .markdownlint.yml CONTRIBUTING.md IMAGES.md README.md
+
+Name:           golang-github-nektos-act
+Release:        %autorelease
+Summary:        None
+
+License:        MIT AND ISC AND BSD-3-Clause AND Apache-2.0 AND BSD-2-Clause
+URL:            %{gourl}
+Source:         %{gosource}
+
+%description %{common_description}
+
+%gopkg
+
+%prep
+%goprep -A
+%autopatch -p1
+
+%if %{without bootstrap}
+%generate_buildrequires
+%go_generate_buildrequires
+%endif
+
+%if %{without bootstrap}
+%build
+for cmd in cmd/* ; do
+  %gobuild -o %{gobuilddir}/bin/$(basename $cmd) %{goipath}/$cmd
+done
+%gobuild -o %{gobuilddir}/bin/act.git %{goipath}
+%endif
+
+%install
+%gopkginstall
+%if %{without bootstrap}
+install -m 0755 -vd                     %{buildroot}%{_bindir}
+install -m 0755 -vp %{gobuilddir}/bin/* %{buildroot}%{_bindir}/
+%endif
```

Comment:
If this package depends on other go packages that are not packaged in Fedora, you probably will need to remove L51-L54, use `%goprep` instead of `%goprep -A`, then add `%define gomodulesmode GO111MODULE=on` right below `%build`, then remove `%gopkginstall` and `%gopkgfiles`.